osbs/build/build_request.py

```diff
@@ -95,6 +98,9 @@ def set_params(self, **kwargs):
 
         self.base_image = kwargs.get('base_image')
         self.low_priority_node_selector = kwargs.get('low_priority_node_selector')
+        if kwargs.get('platform'):
+            self.platform = kwargs.get('platform')
```

Comment:
you need to set platform or else get_platform_nodeselector can't search against it.